ports/cgal/vcpkg.json

```
@@ -67,14 +66,9 @@
       "description": "Qt GUI support for CGAL",
       "dependencies": [
         "eigen3",
-        "qt5-3d",
-        {
-          "name": "qt5-base",
-          "default-features": false
-        },
-        "qt5-script",
-        "qt5-svg",
-        "qt5-xmlpatterns"
+        "qtbase",
```

```suggestion
        {
          "name": "qtbase",
          "default-features": false
        },
```
vcpkg will still install the default features by default, but the user may opt out (CLI, top-level manifest).